core/storage/feedback/gae_models.py

```diff
@@ -179,21 +251,44 @@ def create(cls, exploration_id, thread_id, message_id):
 
     @classmethod
     def get(cls, exploration_id, thread_id, message_id, strict=True):
-        """Gets the FeedbackMessageModel entry for the given ID.
-
-        If the message id is valid and it is not marked as deleted, returns the
-        message instance. Otherwise:
-        - if strict is True, raises EntityNotFoundError
-        - if strict is False, returns None.
+        """Gets the FeedbackMessageModel entry for the given ID. Raises error if
+        no undeleted messages are found and strict == True.
+
+        Args:
+            exploration_id: str. ID of the exploration.
+            thread_id: str. ID of the thread.
+            message_id: int. ID of the message.
+            strict: bool. Whether to raise an error if no FeedbackMessageModel
+                entry is found for the given IDs.
+
+        Returns:
+            FeedbackMessageModel or None. If strict == False and no undeleted
+                message with the given message_id exists in the datastore.
+                Otherwise the FeedbackMessageModel instance that corresponds
+                to the given ID.
+
+        Raises:
+            EntityNotFoundError: If strict == True and message ID is not valid
+                and it is not marked as deleted. No error will be raised if
+                message is deleted.
         """
         instance_id = cls._generate_id(exploration_id, thread_id, message_id)
         return super(FeedbackMessageModel, cls).get(instance_id, strict=strict)
 
     @classmethod
     def get_messages(cls, exploration_id, thread_id):
-        """Returns an array of messages in the thread.
+        """Returns a list of messages in the given thread. The number of
+        messages returned is capped by feconf.DEFAULT_QUERY_LIMIT.
+
+        Args:
+            exploration_id: str. ID of the exploration the thread
+                belongs to.
+            thread_id: str. ID of the thread.
+
+        Returns:
+            list(FeedbackMessageModel). A list of the messages limited by
+                default query limit.
```

Comment:
It would be clearer to explain this "default query limit" thing. How about:

list(FeedbackMessageModel): A list of messages in the given thread, up to a maximum of feconf.DEFAULT_QUERY_LIMIT messages.